sklearn/metrics/_classification.py

```diff
@@ -884,11 +899,25 @@ class labels [2]_.
     sample_weight : array-like of shape (n_samples,), default=None
         Sample weights.
 
+    replace_undefined_by : np.nan, float in [-1.0, 1.0], default=np.nan
+        Sets the return value when a division by zero would occur. This can happen for
```

Comment:
I'd prefer a more generic phrasing such that the param has a common meaning for all metrics
```suggestion
        Sets the return value when the metric is undefined. This can happen for
```